GCD Homework/GCD Homework/Constant.swift

```diff
@@ -11,10 +11,18 @@ class Constant {
     static let getUserURL = "https://api.github.com/search/users?q=abc+in:login"
     
     struct UserTableViewCell {
+        static let identifier = "UserTableViewCell"
```

Comment:
Việc define thế này không sai, tuy nhiên anh có cách tối ưu hơn.
Em tạo một protocol ReuseableCell, define identifier và nibName nhé. Để hiểu kỹ hơn implement thế nào thì đây nha
https://medium.com/@gonzalezreal/ios-cell-registration-reusing-with-swift-protocol-extensions-and-generics-c5ac4fb5b75e